src/middlewares/Validate.js

```diff
@@ -67,5 +48,27 @@ class ValidateUser {
       check('startDate', 'date should be validate like in this format(YYYY-DD--MM)').isBefore(startDates)
     ];
   }
+
+  /**
+* Validate input
+* @static
+* @returns {object} errors
+*/
+  static profileUpdateRules() {
+    const d = new Date();
+    const year = d.getFullYear();
+    const month = d.getMonth();
+    const day = d.getDate();
+    const minAge = new Date(year - 16, month, day).toDateString();
+    return [
+      check('gender', 'Gender should be either Male or Female').isAlpha(),
```

Comment:
Make sure if it validates only whether male or female unless the user can insert anything that is not valid. if this not working, make research on `checkif` is one of the express validators. This can help.